drivers/gpio/gpio_b9x.c

```diff
@@ -119,6 +122,12 @@ struct gpio_b9x_config {
 struct gpio_b9x_data {
 	struct gpio_driver_data common; /* driver data */
 	sys_slist_t callbacks;          /* list of callbacks */
+	struct gpio_b9x_t gpio_b9x_periph_config;
```

Comment:
Create separate structure to encapsulate gpio backup.
gpio_b9x_backup

Reply:
done